lib/stripe_mock/request_handlers/helpers/subscription_helpers.rb

```diff
@@ -111,7 +111,31 @@ def verify_trial_end(trial_end)
 
       def total_items_amount(items)
         total = 0
-        items.each { |i| total += (i[:quantity] || 1) * i[:plan][:amount] }
+        items.each do |item|
+          if not item[:plan][:amount].nil?
```

Comment:
we don't follow ruby style guide yet, but can you run rubocop locally exactly for your changes?
Also can it be modified with using `unless` and `next` commands to avoid constructions like `if not something.nil?`